scripts/sct_label_utils.py

```diff
@@ -598,6 +602,30 @@ def launch_sagittal_viewer(self, labels):
 
         return output
 
+    def remove_or_keep_labels(self, labels, action):
+        """
+        Create or remove labels from self.image_input
+        :param list(int): Labels to add or remove
```

Comment:
i would replace "add" by "keep"

Reply:
Oops my mistake. Changes made in commit [0fe767d](https://github.com/neuropoly/spinalcordtoolbox/commit/0fe767df0718e84e21a26f1ca21a444364431075)